Makefile

```diff
@@ -5,6 +5,8 @@ setup: install-deps launch-registry
 install-deps:
 	npm install bats@1.x
 	docker pull busybox
+	command -v shellcheck || echo "  please apt/brew install shellcheck"
```

Comment:
not super reproducible, but so far have been relying on the OS-installed one

Reply:
Given the scope of this repo (currently two scripts) I would propose that we merge this nonetheless. We also are not installing npm or docker with the makefile.

Reply:
Practically the rate of change on shellcheck is also quite low, [one or two versions per year](https://github.com/koalaman/shellcheck/tags). Approved.